dataactcore/interfaces/function_bag.py

```diff
@@ -647,3 +647,24 @@ def get_last_validated_date(submission_id):
     # Still need to do a check here in case there aren't any jobs for a submission.
     # This is the case for a single unit test
     return oldest_date.strftime('%m/%d/%Y') if oldest_date else oldest_date
+
+
+def get_action_dates(submission_id, valid=True):
+    """ Pull the earliest/latest action dates from the DetachedAwardFinancialAssistance table
+
+    Arguments:
+        submission_id: submission ID pull action dates from
+        valid: true if we should only go over valid action dates
+
+    Returns:
+        the earliest action date (str)
+        the latest action date (str)
+    """
+
+    sess = GlobalDB.db().session
+    action_dates = sess.query(func.min(DetachedAwardFinancialAssistance.action_date).label("min_action_date"),
+                              func.max(DetachedAwardFinancialAssistance.action_date).label("max_action_date"))\
+        .filter(DetachedAwardFinancialAssistance.submission_id == submission_id,
+                DetachedAwardFinancialAssistance.is_valid == valid)
```

Comment:
I don't think comparing true/false this way is accurate (or correct at least). You should be using `DetachedAwardFinancialAssistance.is_valid.is_(True)`. I also specify `True` because at no point will we be looking for `False` to my knowledge so this probably doesn't need to be a variable either.